docs/example.ipynb

```diff
@@ -179,7 +190,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.12.1"
```

Comment:
Not sure if this will cause an issue, but since change it back to 3.9 just in case.
```suggestion
   "version": "3.9.18"
```